Background Color Changer/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>

```

Comment:
- Write a folder name  `Background_Color_Changer` not this `Background Color Changer`